src/dotnet-core-uninstall/Shared/Commands/UninstallCommandExec.cs

```diff
@@ -241,17 +215,18 @@ public static bool AskWithWarningsForRequiredBundles(IDictionary<Bundle, string>
                 Console.Write(string.Format(RuntimeInfo.RunningOnWindows ? LocalizableStrings.WindowsRequiredBundleConfirmationPromptOutputFormat :
                     LocalizableStrings.MacRequiredBundleConfirmationPromptOutputFormat, pair.Key.DisplayName, pair.Value));
                 Console.ResetColor();
-                var response = userResponse == null ? Console.ReadLine().Trim().ToUpper() : userResponse.ToUpper();
+                
+                var response = userResponse?.ToUpper() ?? Console.ReadLine().Trim().ToUpper();
+
                 if (response.Equals("N"))
```

Comment:
Should we be checking for `NO` as well, given that we check for `y` or `yes`? If we really want to restrict an interactive response, then I wonder if a loop with ReadKey is better and we only accept N/n/y/Y

Reply:
Even better.

Reply:
Updated, let me know what you think :)